google-cloud-bigtable/lib/google/cloud/bigtable/chunk_reader.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
 module Google
   module Cloud
     module Bigtable
-      # Invalid row state error
-      class InvalidRowStateError < Google::Cloud::Error
```

Comment:
Should this file now `require "google/cloud/bigtable/errors"`?

Reply:
Made the change.